zul/src/main/resources/web/zul/less/font/_core.less

```diff
@@ -15,6 +15,7 @@
 .@{fa-css-prefix}-regular,
 .fab,
 .@{fa-css-prefix}-brands {
+  // DO NOT modify this part, maintain this settings of FontAwesome 4.7.0, otherwise it'll cause B85-ZK-3800 side effect
```

Comment:
To be more flexible, we should not "forbidden" developers to change the code.
Just "NOTICE that the code is to match...."
For later maintaining, developers should test it